config/typeorm.config.ts

```diff
@@ -3,6 +3,9 @@ import * as dotenv from 'dotenv';
 
 dotenv.config();
 
+const migrationsFolder = process.env.MIGRATIONS_TYPE === 'seed' ? `${__dirname}/../seeds/*{.ts,.js}` : `${__dirname}/../migrations/*{.ts,.js}`
```

Comment:
Let's try to move using env variable to the script 
Crate 2 scripts: 1 for migration and 1 for seed